<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrowBookInformation</class>
 <widget class="QWidget" name="borrowBookInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="searchpushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Êü•ËØ¢üîç</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>89</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>‰π¶Âêç</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bookNameSearchLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Âá∫ÁâàÁ§æ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pressSearchLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>‰ΩúËÄÖ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="writerSearchLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>861</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ÂÄüÈòÖÂõæ‰π¶‰ø°ÊÅØ</string>
   </property>
  </widget>
  <widget class="QTableView" name="returnedBookInformationtableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>851</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AllpushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ÂÖ®ÈÉ®</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
